src/Components/Organisms/Navigation/navigation.js

```diff
@@ -1,36 +1,70 @@
 import "./navigation.scss";
 import jsonFile from "src/db.json";
 import React, { useState, useRef, useEffect } from "react";
+import Menu from "src/Components/Molecules/menu/menu";
+import SubMenu from "src/Components/Molecules/sub-menu.js/sub-menu";
 import CartDropDown from "src/Components/Molecules/Cart-drop-down/cart-drop-down";
 import Hamburger from "src/Components/Atoms/buttons/hamburger-button/hamburger";
 
 const Navigation = () => {
     const navigation = jsonFile.navigation;
 
-    const [isopenCart, setOpenCart] = useState(false);
+    const [isOpenCart, setOpenCart] = useState(false);
+    const [isMenuActive, setMenuActive] = useState(false);
```

Comment:
isMenuActive is for opening main menu?
Could have done the same pattern as for other hooks and named it like isMainMenuOpen